### PR TITLE
Add helm chart

### PR DESCRIPTION
**Pull request by hlascelles**
This adds a helm chart for this deployment. Values are: 

```
enabled: true
image:
  repository: x0rg/kube-nodes-downscaler
  tag: v0.0.5
  pullPolicy: IfNotPresent
namespace: kube-system
args: []
```

Note, until this is merged to master, the `index.yaml` must point the available `tgz` on the `bambooengineering` branch for us to use it.

### CODE REVIEW

**Review by Raffo — APPROVED** [review]
Thanks for your contribution, this looks good to me! 
I only need to change https://github.com/Raffo/kube-nodes-downscaler/pull/8/files#diff-1433bd115089a2c52cc32951548f6552R10 from `bambooengineering` to `Raffo`, right? 

EDIT: I will wait for the answer before merging such that I can immediately update the reference after.

**Comment by hlascelles** [comment]
You'll need to change the branch to `master` too. I believe it will become:

`https://raw.githubusercontent.com/Raffo/kube-nodes-downscaler/master/charts/kube-nodes-downscaler-0.1.0.tgz`

**Pull request merged**
